components/brave_welcome_ui/containers/screens/importBox.tsx

```diff
@@ -72,7 +72,7 @@ export default class ImportBox extends React.PureComponent<Props, State> {
             >
               <option key={0} value=''>{getLocale('importFrom')}</option>
               {
-                isValidBrowserProfiles
+                isValidBrowserProfiles(browserProfiles)
```

Comment:
note that this was always returning true before since `isValidBrowserProfiles` is a defined function